.chloggen/enhance-agent-daemonset-toleration.yaml

```diff
@@ -0,0 +1,12 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+# The name of the component, or a single word describing the area of concern, (e.g. agent, clusterReceiver, gateway, operator, chart, other)
+component: agent
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Make it so the default tolerations used to deploy the agent collector account for k8s distribution
+# One or more tracking issues related to the change
+issues: [1562]
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext: OpenShift infra nodes and Azure system nodes will now be monitored by the agent by default
```

Comment:
is this correct? It appears we explicitly set to not schedule by default on openshift and azure.
https://github.com/signalfx/splunk-otel-collector-chart/pull/1562/files#diff-a196d36724bc1a2c08c5c54b9c3efe8763e29bb0c007995455b93ab280710c8fR87

Reply:
The changelog is correct. Tolerations on pods work with taints on nodes. K8s will schedule a pod on a node with a taint like `key=value:NoSchedule` only if the pod tolerates it. That's what Josh's change does, it tells k8s that the agent pod tolerate these node taints and to schedule it anyway.

Reply:
ok thanks!